projects/arc-docs/src/app/docs/docs.component.scss

```diff
@@ -1,3 +1,71 @@
+@use 'sass:map';
+@use '../../../../arc-lib/src/lib/theme/styles/variables' as *;
```

Comment:
add this in angular json 
"stylePreprocessorOptions": {
              "includePaths": [
                "src/app/theme"
              ]
            }, and use it like @import 'element-override'; in scss files